tests/FunctionsTest.php

```diff
@@ -67,18 +67,20 @@ public function testCaptureException(): void
 
     public function testCaptureEvent(): void
     {
-        $eventId = EventId::generate();
+        $event = Event::createEvent($eventId = EventId::generate());
+
+        $event->setMessage('foo');
```

Comment:
I think you can remove this line, we were capturing the message because it was the only way to know if the event was being created correctly but we can now compare the instances